packages/core/src/Layout/Layout.ts

```diff
@@ -75,11 +104,13 @@ class Layout {
     this.scriptBottom = nodeProcessor.scriptBottom;
 
     pageSources.addAllToSet(this.includedFiles);
-    await this.config.externalManager.generateDependencies(pageSources.getDynamicIncludeSrc(),
-                                                           this.includedFiles);
+    await this.config.externalManager.generateDependencies(
+      pageSources.getDynamicIncludeSrc(),
+      this.includedFiles,
+    );
```

Comment:
Is this also an unnecessary whitespace change?

Reply:
this exceeds the max line length, so i think this one is ok

Reply:
iirc our max line length is 110? The existing one is fine by my linter

Reply:
ok yeah, i've reverted it to the original now